package.json

```diff
@@ -60,4 +64,4 @@
     "last 2 versions",
     "not dead"
   ]
-}
+}
```

Comment:
new line at the end of the file. Probably should fix IDE config to prevent future occurances.